src/grids/PackingGrid.ts

```diff
@@ -58,8 +57,8 @@ export interface PackingGridOptions extends GridOptions {
 }
 
 /**
- * The PackingGrid is a grid that shows the important items bigger without sacrificing the weight of the items. Rows and columns are separated so that items are dynamically placed within the horizontal and vertical space rather than arranged in an orderly fashion.
- * @ko PackingGrid는 아이템의 본래 크기에 따른 비중을 해치지 않으면서 중요한 카드는 더 크게 보여 주는 레이아웃이다. 행과 열이 구분돼 아이템을 정돈되게 배치하는 대신 가로세로 일정 공간 내에서 동적으로 아이템을 배치한다.
+ * The PackingGrid is a grid that shows the important items bigger without sacrificing the weight of the items. Rows and columns are separated so that items are dynamically placed within the horizontal and vertical space rather than arranged in an orderly fashion. The position and size of the items are different depending on the values of the `sizeWeight` and `ratioWeight` options.
+ * @ko PackingGrid는 아이템의 본래 크기에 따른 비중을 해치지 않으면서 중요한 카드는 더 크게 보여 주는 레이아웃이다. 행과 열이 구분돼 아이템을 정돈되게 배치하는 대신 가로세로 일정 공간 내에서 동적으로 아이템을 배치한다. `sizeWeight`와 `ratioWeight` 옵션 값에 따라 아이템이 배치되는 위치, 사이즈가 달라진다.
```

Comment:
Additional explanations are needed to help users predict how importance is used and how importance is affected.